app/debt/save-debt-data.js

```diff
@@ -1,7 +1,8 @@
 const db = require('../data')
 
 const saveDebtData = (updatedDebtData, transaction) => {
-  const { paymentRequestId, attachedDate, debtDataId } = updatedDebtData
+  const { paymentRequestId, debtDataId } = updatedDebtData
```

Comment:
Any justification for not retrieving attachedDate from updatedDebtData as it was before?

Reply:
Didnt realise I could retrive it and thought it needed redeclaring. I will amend.

Reply:
resolved as discussed by renaming `updatedDebtData` to `debtData` for clarity